main/src/ca/uwaterloo/flix/Main.scala

```diff
@@ -103,6 +103,7 @@ object Main {
       xfuzzer = cmdOpts.xfuzzer,
       xprinttyper = cmdOpts.xprinttyper,
       xverifyeffects = cmdOpts.xverifyeffects,
+      xverifylocations = cmdOpts.xverifylocations,
```

Comment:
magnus wanted one flag for all the verifiers, so maybe just call this `xverify`?